go.mod

```diff
@@ -3,34 +3,46 @@ module github.com/kubernetes-sigs/nfs-subdir-external-provisioner
 go 1.14
 
 require (
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
-	k8s.io/kubernetes v1.18.0
+	github.com/golang/glog v1.0.0
+	k8s.io/api v0.23.4
+	k8s.io/apimachinery v0.23.4
+	k8s.io/client-go v0.23.4
+	k8s.io/kubernetes v1.23.4
 	sigs.k8s.io/sig-storage-lib-external-provisioner/v6 v6.0.0
 )
 
 replace (
-	k8s.io/api => k8s.io/api v0.18.0
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.0
-	k8s.io/apimachinery => k8s.io/apimachinery v0.18.2-beta.0
-	k8s.io/apiserver => k8s.io/apiserver v0.18.0
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.18.0
-	k8s.io/client-go => k8s.io/client-go v0.18.0
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.18.0
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.18.0
-	k8s.io/code-generator => k8s.io/code-generator v0.18.3-beta.0
-	k8s.io/component-base => k8s.io/component-base v0.18.0
-	k8s.io/cri-api => k8s.io/cri-api v0.18.11-rc.0
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.18.0
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.0
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.18.0
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.18.0
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.18.0
-	k8s.io/kubectl => k8s.io/kubectl v0.18.0
-	k8s.io/kubelet => k8s.io/kubelet v0.18.0
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.18.0
-	k8s.io/metrics => k8s.io/metrics v0.18.0
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.18.0
+	k8s.io/api => k8s.io/api v0.23.4
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.23.4
+	k8s.io/apimachinery => k8s.io/apimachinery v0.23.4
+	k8s.io/apiserver => k8s.io/apiserver v0.23.4
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.23.4
+	k8s.io/client-go => k8s.io/client-go v0.23.4
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.23.4
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.23.4
+	k8s.io/code-generator => k8s.io/code-generator v0.23.4
+	k8s.io/component-base => k8s.io/component-base v0.23.4
+	k8s.io/cri-api => k8s.io/cri-api v0.23.4
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.23.4
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.23.4
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.23.4
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.23.4
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.23.4
+	k8s.io/kubectl => k8s.io/kubectl v0.23.4
+	k8s.io/kubelet => k8s.io/kubelet v0.23.4
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.23.4
+	k8s.io/metrics => k8s.io/metrics v0.23.4
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.23.4
 )
+
+replace k8s.io/component-helpers => k8s.io/component-helpers v0.23.4
```

Comment:
nit: it would be nicer to have these replace statements in the replace block above.

Reply:
sure, done